src/lib/converter/plugins/ImplementsPlugin.ts

```diff
@@ -38,6 +38,9 @@ export class ImplementsPlugin extends ConverterComponent
             }
 
             var classMember:DeclarationReflection;
+
+            if (!classReflection.children) return;
```

Comment:
This should do the trick. The style does seem inconsistent though. I think the return should either be on a new line (like the `if (!classMember)`), or wrap the for loop.